insertion-sort/readme.md

```diff
@@ -0,0 +1,3 @@
+# Insertion Sort
+
```

Comment:
You need to explain what insertion sort is and how it works; you can even walk through it with your drawing